ansible/nats/install.yml

```diff
@@ -0,0 +1,283 @@
+---
+- name: Add nats helm support
+  hosts: localhost
+  connection: local
+  vars:
+    ns: qa
+    nats_cmd_pod: nats-box
+    ui_wildcard_fqdn: "*.cyverse.org"
+    vice_wildcard_fqdn: "*.cyverse.run"
+    vice_base_domain: cyverse.run
+    ca_duration: 8766h # 1 year
+    ca_renew_before: 240h # 10 days
+    nats_server_cert_duration: 8766h # 1 year
+    nats_server_cert_renew_before: 240h # 10 days
+    nats_client_cert_duration: 8766h # 1 year
+    nats_client_cert_renew_before: 240h # 10 days
```

Comment:
I had the same question about the play vars here. Is there a way to override them with inventory group vars?